packages/teleterm/electron-builder-config.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @type { import('electron-builder').Configuration }
+ */
```

Comment:
This doesn't seem to work (just as other typings provided through JSDoc in other files in our project) but I left it here anyway in case we manage to fix it in the future.

Reply:
As we talked, it works only for autocomplete:

<img width="600" alt="image (2)" src="https://user-images.githubusercontent.com/20583051/184882694-acc9f8dd-0dbb-402e-a816-cc99705ed919.png">